examples/scotch_dashboard.ipynb

```diff
@@ -34,7 +34,11 @@
       "version": 1,
       "views": {
        "grid_default": {
-        "hidden": true
+        "col": 0,
+        "height": 2,
+        "hidden": true,
+        "row": 29,
+        "width": 12
```

Comment:
Just realized the extra metadata fields are added to the notebook even when `hidden` is set to `true`.

Reply:
Yes, I do the same in `jupyterlab-gridstack` to prevent errors.

Reply:
By "errors" you mean the extension expects these fields to be in the metadata also when `hidden` is `true`?

Reply:
Just checked the spec: https://jupyter-dashboards-layout.readthedocs.io/en/latest/metadata.html

And there doesn't seem to be anything regarding whether other fields should be present when `"hidden": true`. So I guess we can decide whether to keep them or not.

Reply:
Then we can remove it when `"hidded":true` and we have fewer data in the notebook's metadata.

Reply:
Pushed https://github.com/voila-dashboards/voila-gridstack/pull/67/commits/60961a97368d0c98d9ddde5663f79f22cf32f01d to handle this, so the classic notebook extension behaves as before for now.